share/completions/sv.fish

```diff
@@ -10,12 +10,19 @@ set -l commands \
     shutdown force-stop force-reload force-restart force-shutdown \
     try-restart check
 
+set -l search_svdirs \
+    /run/runit/runsvdir/current \
+    /run/runit/service \
+    /etc/services \
+    /services \
+    #
```

Comment:
> Finally return quietly if there is no svdir.

I don't think this actually works.
Note that the # doesn't do anything. You probably wanted to write "" here.

this alternative approach would also work 

```fish
set -l svdir
for dir in \
    "$SVDIR" \
    /run/runit/runsvdir/current \
    /run/runit/service \
    /etc/services \
    /services
    if test -d $dir
        set svdir $dir
        break
    end
end
set -q svdir[1]; or return
```

Note that we actually only need to quote $SVDIR (for edge cases), so doing that explicitly seems better.

Reply:
> Note that the # doesn't do anything. You probably wanted to write "" here.

It was on purpose, to make sure all lines on the list ended with `\`, but I realize it's stupid.

> ```fish
> test -d $dir
> ```
> Note that we actually only need to quote $SVDIR (for edge cases), so doing that explicitly seems better.

We also need to quote variables in `test`, see faho's comment, I had no idea about this quirk with `test` until now.